Jenkinsfile.dependency

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env groovy
+// This shared library is available at https://github.com/ROCmSoftwarePlatform/rocJENKINS/
+@Library('rocJenkins') _
+
+// This is file for internal AMD use.
+// If you are interested in running your own Jenkins, please raise a github issue for assistance.
+
+import com.amd.project.*
+import com.amd.docker.*
+
+////////////////////////////////////////////////////////////////////////
+// Mostly generated from snippet generator 'properties; set job properties'
+// Time-based triggers added to execute nightly tests, eg '30 2 * * *' means 2:30 AM
+properties([
+    pipelineTriggers([cron('0 1 * * *'), [$class: 'PeriodicFolderTrigger', interval: '5m']]),
```

Comment:
Should this not include the PeriodicFolderTrigger so we only run nightly?

Also, looking at the help for this suggests we should be using the "H 1 * * *" instead of "0 1 * * *" so the jobs don't all start at once.

Reply:
I removed cron, and just going on a folder trigger so it happens only if Tensile changes.

Reply:
@sdquiring Also, feedback on install.sh file is going to go in another PR.